static/app/components/replays/replayController.tsx

```diff
@@ -89,7 +88,7 @@ function ReplayPlayPauseBar({isCompact}: {isCompact: boolean}) {
             if (!startTimestampMs) {
               return;
             }
-            const transformedCrumbs = transformCrumbs(replay?.getRawCrumbs() || []);
```

Comment:
Does TS not yell?

Reply:
`transformCrumbs` accepts a list of `RawCrumb`, and returns a list of `Crumb`.

`Crumb` is a superset of `RawCrumb`, so TS doesn't complain

```
export type Crumb = RawCrumb & {
  color: Color;
  description: string;
  id: number;
};
```